docs/tutorials/mongoid-upgrade.txt

```diff
@@ -280,6 +280,61 @@ Example Mongoid 7.1 behavior:
 Mongoid provides :ref:`sharding management Rake tasks <sharding-management>`
 to shard collections according to shard keys declared in models.
 
+Changing the Discriminator Key
+------------------------------
+
+Mongoid now supports changing the default discriminator key from the default ``_type`` when using
```

Comment:
Can you please also link to the section of the manual on the document page for users to follow to learn more about discriminator key/value overrides. This is similar to how other sections are linked.